### PR TITLE
Update concourse notifications to send to Google Chat Spaces

### PR DESCRIPTION
**Pull request by axue-broadcom**

### CODE REVIEW

**Review by bradfordb-vmware — APPROVED** [review]

**Review by tatb-vmware — APPROVED** [review]

File: concourse/pipelines/hadoop-cluster-cleaner.yml [hunk 1]
```diff
@@ -25,17 +25,17 @@ resources:
     branch: ((pxf-git-branch))
     uri: ((ud/pxf/common/git-remote))
 
-- name: slack-alert
+- name: gchat-alert
   type: slack-notification
```
Comment (denalex) [inline concourse/pipelines/hadoop-cluster-cleaner.yml:29]: needs name update for the resource type ?

File: concourse/pipelines/hadoop-cluster-cleaner.yml [hunk 2]
```diff
@@ -25,17 +25,17 @@ resources:
...
   source:
-    url: ((ud/pxf/secrets/ud-pipeline-bot-ud-dev-webhook))
+    url: ((ud/pxf/secrets/ud-pipeline-bot-gp-ud-gchat-webhook))
```
Comment (denalex) [inline concourse/pipelines/hadoop-cluster-cleaner.yml:31]:
```suggestion
    url: ((ud/pxf/secrets/gchat-gp-extended-federation-ci-webhook))
```
the "CI" webhook for a given GChat space

**Review by yimingli-vmware — APPROVED** [review]

File: concourse/scripts/generate_gchat_message.bash [hunk 3]
```diff
@@ -7,8 +7,8 @@ set -e
 : "${RELENG_DROP_URL:?RELENG_DROP_URL must be set}"
 : "${RELENG_OSL_DROP_URL:?RELENG_OSL_DROP_URL must be set}"
 : "${RELENG_ODP_DROP_URL:?RELENG_ODP_DROP_URL must be set}"
-: "${PXF_SLACK_CHANNEL_NAME:?PXF_SLACK_CHANNEL_NAME must be set}"
-: "${PXF_SLACK_CHANNEL_LINK:?PXF_SLACK_CHANNEL_LINK must be set}"
+: "${PXF_GOOGLE_SPACE_NAME:?PXF_GOOGLE_SPACE_NAME must be set}"
+: "${PXF_GOOGLE_SPACE_LINK:?PXF_GOOGLE_SPACE_LINK must be set}"
```
Comment (denalex) [inline concourse/scripts/generate_gchat_message.bash:11]:
```suggestion
: "${PXF_GCHAT_SPACE_NAME:?PXF_GCHAT_SPACE_NAME must be set}"
: "${PXF_GCHAT_SPACE_LINK:?PXF_GCHAT_SPACE_LINK must be set}"
```

File: concourse/scripts/generate_gchat_message.bash [hunk 4]
```diff
@@ -40,11 +40,11 @@ pxf_gp7_el9_releng_url="${RELENG_DROP_URL}/gpdb7/pxf-gp7-${version}-2.el9.x86_64
 pxf_gp7_osl_releng_url="${RELENG_OSL_DROP_URL}/gpdb7/${PXF_OSL_FILE_PREFIX}_${version}_GA.txt"
 pxf_gp7_odp_releng_url="${RELENG_ODP_DROP_URL}/gpdb7/${PXF_ODP_FILE_PREFIX}-${version}-ODP.tar.gz"
 
-echo "Generating Slack Message"
+echo "Generating Google Space Message"
```
Comment (denalex) [inline concourse/scripts/generate_gchat_message.bash:43]:
```suggestion
echo "Generating Google Chat Space Message"
```

File: concourse/scripts/generate_gchat_message.bash [hunk 5]
```diff
@@ -72,9 +72,8 @@ The GPDB7 artifacts are:
...
 
 Can you please upload the artifacts and the OSL / ODP files to the Greenplum Tanzu Network Release for our product, PXF?
-The OSL file should appear as "Open Source Licenses for PXF ${version}".
-
-Once the artifacts are published to the Tanzu Network site, please post a message in the #${PXF_SLACK_CHANNEL_NAME} slack channel:
-${PXF_SLACK_CHANNEL_LINK}
+The OSL file should appear as \"Open Source Licenses for PXF ${version}\".
 
+Once the artifacts are published to the Tanzu Network site, please post a message in the #${PXF_GOOGLE_SPACE_NAME} Google Space:
+${PXF_GOOGLE_SPACE_LINK}
```
Comment (denalex) [inline concourse/scripts/generate_gchat_message.bash:78]:
```suggestion
Once the artifacts are published to the Tanzu Network site, please post a message in the #${PXF_GCHAT_SPACE_NAME} Google Space:
${PXF_GCHAT_SPACE_LINK}
```

File: concourse/tasks/generate_gchat_message.yml [hunk 6]
```diff
@@ -14,11 +14,11 @@ outputs:
 params:
   PXF_OSL_FILE_PREFIX:
   PXF_ODP_FILE_PREFIX:
-  PXF_SLACK_CHANNEL_NAME:
-  PXF_SLACK_CHANNEL_LINK:
+  PXF_GOOGLE_SPACE_NAME:
+  PXF_GOOGLE_SPACE_LINK:
```
Comment (denalex) [inline concourse/tasks/generate_gchat_message.yml:18]:
```suggestion
  PXF_GCHAT_SPACE_NAME:
  PXF_GCHAT_SPACE_LINK:
```

**Review by denalex — APPROVED** [review]

**Pull request merged**